hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/protocolPB/CommonPBHelper.java

```diff
@@ -0,0 +1,135 @@
+/**
```

Comment:
Currently, PBHelper is used in Hadoop, and all methods in this class should be putted in PBHelper.
Does this patch for the old version of Hadoop?